.eslintrc.json

```diff
@@ -0,0 +1,14 @@
+{
+	"extends": ["@financial-times/eslint-config-next"],
+	"parserOptions": {
+		"ecmaVersion": 2020
```

Comment:
nickpick: Just wondering about the es version. Why not 2022? Just

Reply:
We're currently supporting Node.js 14 and 16 (because we have apps running on both) and based on https://node.green/ it looks like Node.js 14 doesn't have particularly good ES2021 or ES2022 support.